inspire/base/searchext/mappings/hep.json

```diff
@@ -121,7 +121,8 @@
                     "type": "string"
                 },
                 "authors": {
-                    "type": "object",
+                    "type": "nested",
+                    "include_in_parent": true,
```

Comment:
So I need ```nested``` to use my queries and ```"include_in_parent": true``` allows nested type to behave like an object at the same time (during testing, something is accessing authors like it is an object, so this trick necessary to keep everything working).